mods/ra2/sequences/allied-infantry.yaml

```diff
@@ -52,6 +52,8 @@ e1:
 	Inherits: ^ArmedInfantry
 	Inherits@MC: ^MindControllable
 	Defaults: gi
+	dot: dot
+		ZOffset: -9999
```

Comment:
This looks like a workaround/hack. What speaks against adding the `WithDeployingInfantryBody` suggested on IRC?